### PR TITLE
[ACA-2654] show ellipsis for long Size column headers

### PR DESCRIPTION
**Pull request by suzanadirla**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [x] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[x] Other... Please describe: UI changes
```

## What is the current behavior?

Issue Number: https://issues.alfresco.com/jira/browse/ACA-2654

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/app/ui/overrides/adf-document-list.theme.scss [hunk 1]
```diff
@@ -58,6 +58,16 @@
 
       &.adf-datatable-cell--fileSize {
         min-width: 80px;
+
+        span {
```
Comment (DenysVuika) [inline src/app/ui/overrides/adf-document-list.theme.scss]: Can you move this to ADF please?

Comment (suzanadirla) [reply src/app/ui/overrides/adf-document-list.theme.scss]: Moved ;) https://github.com/Alfresco/alfresco-ng2-components/pull/4902 and ready for review

Comment (DenysVuika) [reply src/app/ui/overrides/adf-document-list.theme.scss]: If it’s in the ADF with that PR, we don’t need to duplicate the styles in our code

Comment (suzanadirla) [reply src/app/ui/overrides/adf-document-list.theme.scss]: The extra ADF styles will be removed with [ACA-2361](https://issues.alfresco.com/jira/browse/ACA-2361) which is waiting for the upgrade to 3.4.0 ADF

Comment (suzanadirla) [reply src/app/ui/overrides/adf-document-list.theme.scss]: but, you are right, I'll remove the extra styles now

**Pull request merged**